common/provision.sh

```diff
@@ -0,0 +1,63 @@
+
+function print_params() {
+
+    echo "## print common parameters"
+
+    declare -A params=( ["VERSION"]=$VERSION ["PRERELEASE"]=$PRERELEASE ["BUILD"]=$BUILD \
+                        ["CORE_TAG_NAME"]=$CORE_TAG_NAME ["PLUGINS_TAG_NAME"]=$PLUGINS_TAG_NAME )
+    for param in "${!params[@]}"
+    do
+            echo "$param - ${params["$param"]}"
+    done
+}
+
+function install_common_prereqs
```

Comment:
please use function declarations as in other functions `function x() {}`

thank